test/stores/memory.unit.js

```diff
@@ -37,6 +37,23 @@ describe("memory store", function() {
         });
     });
 
+    describe("ttl", function() {
+        var memoryCache;
+
+        beforeEach(function() {
+            memoryCache = memoryStore.create({noPromises: true, ttl: 0.001});
+        });
+
+        it("if options arg is a number in set()", function(done) {
+            memoryCache.set('foo', 'bar', 60);
+
+            setTimeout(function() {
+                assert.equal(memoryCache.get('foo'), 'bar');
+                done();
```

Comment:
Maybe add another test here to make sure the cache record expires after the TTL?